app/gui/src/dashboard/App.tsx

```diff
@@ -399,11 +399,6 @@ function AppRouter(props: AppRouterProps) {
 
   const mainPageUrl = getMainPageUrl()
 
-  // Subscribe to `localStorage` updates to trigger a rerender when the terms of service
-  // or privacy policy have been accepted.
-  localStorageProvider.useLocalStorageState('termsOfService')
```

Comment:
what is this superseded by?

Reply:
Good question. 

_Before_ react compiler we used to use this line to rerender the react tree if `termsOfService` changes, but after adding it (obviously) stopped working, so I made a fix: https://github.com/enso-org/enso/pull/11473

In this PR I completely removed the subscription as it no longer affects anything - compiler skips rendering of all nested elements. 

I don't think this line would affect anything (We render the `AgreementsModal` anyway and it has it's own subscription)